packages/hint-typescript-config/src/is-valid.ts

```diff
@@ -39,12 +39,15 @@ export default class TypeScriptConfigIsValid implements IHint {
         };
 
         const invalidSchema = async (fetchEnd: TypeScriptConfigInvalidSchema) => {
-            const { prettifiedErrors, resource } = fetchEnd;
+            const { errors, prettifiedErrors, resource } = fetchEnd;
 
             debug(`parse::typescript-config::error::schema received`);
 
-            for (const error of prettifiedErrors) {
-                await context.report(resource, null, error);
+            for (let i = 0; i < errors.length; i++) {
```

Comment:
Why a `for` instead of the `for.. of`? I'm just curious.

Reply:
I needed an index to access the corresponding items from both `prettifiedErrors` and `errors` in order to get the message text from the former and `ProblemLocation` from the latter.

Reply:
That make sense hehehe, I miss the part where you use the index hehehe.